hack/generate/samples/internal/go/v3/memcached_with_webhooks.go

```diff
@@ -92,6 +92,13 @@ func (mh *MemcachedGoWithWebhooks) Run() {
 	mh.implementingWebhooks()
 	mh.uncommentKustomizationFile()
 
+	sampleFile := filepath.Join("config", "samples",
+		fmt.Sprintf("%s_%s_%s.yaml", mh.ctx.Group, mh.ctx.Version, strings.ToLower(mh.ctx.Kind)))
+
+	log.Infof("updating sample to have size attribute")
+	err = testutils.ReplaceInFile(filepath.Join(mh.ctx.Dir, sampleFile), "foo: bar", "size: 1")
+	pkg.CheckError("updating sample", err)
```

Comment:
Nit: is this any different from the code in v2 sample code? If not can you refactor these lines into `implementingAPI()`?